install.rst

```diff
@@ -5,6 +5,44 @@ Install
 Choose the install method that makes sense for you.
 
 
+Prerequisites
+==================
+
+Server
+-----------------
+
+All components of Netmaker can be run on a single server (Virtual Machine or Bare Metal). Here some recommendations for setting up the server:
+
+- We **highly recommend** that Netmaker be deployed in a dedicated networking environment. It should not share a local network with the clients which it will be managing. This can cause routing issues.
```

Comment:
remove the `It should not share a local network with the clients which it will be managing. This can cause routing issues.`